src/core/createEvent.js

```diff
@@ -58,6 +58,20 @@ export default () => {
     documentMayUnload() {
       documentMayUnload = true;
     },
+    augmentEvent(callback, extraOptions) {
```

Comment:
Since augmentEvent is never called outside of this file, you can defined it locally and not expose it.

Reply:
ok, do you think we should revert this at all?

Reply:
either way. You could go back to how it was before or define this as a local function.